main.js

```diff
@@ -1304,6 +1347,17 @@ installSettingsSetter('incoming-call-notification');
 
 // These ones are different because its single source of truth is userConfig,
 // not IndexedDB
+ipc.on('get-stay-in-background', event => {
+  event.sender.send(
+    'get-success-stay-in-background',
+    null,
+    userConfig.get('stayInBackground') || false
```

Comment:
The default value for stayInBackground is true based on the current implementation, if there is no value yet the getter should return the default which is true.

Reply:
So the current behaviour is that the  `config` value remains undefined until users close the window the first time.

Two cases can occur:
1) 
User goes in settings first, select `Stay in background` option. 

Then if user closes window, prompt notification will not show since user already set the value to `true` or `false`

2) 
User close the window without going to the settings. While closing the window, the prompt notification is shown, and `config` is set to `true`

Then if user goes in settings value will be `true`

In order to keep this behaviour, ill update it to return `null` explicitly when `config` is not `true` or `false`